auth0/src/test/java/com/auth0/android/request/internal/AuthenticationErrorBuilderTest.java

```diff
@@ -0,0 +1,73 @@
+package com.auth0.android.request.internal;
+
+import com.auth0.android.Auth0Exception;
+import com.auth0.android.authentication.AuthenticationException;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.any;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.junit.Assert.assertThat;
+
+public class AuthenticationErrorBuilderTest {
+
+    private AuthenticationErrorBuilder builder;
+
+    @Before
+    public void setUp() throws Exception {
+        builder = new AuthenticationErrorBuilder();
+    }
+
+    @Test
+    public void shouldCreateFromMessage() throws Exception {
+        final AuthenticationException ex = builder.from("message");
+
+        assertThat(ex.getCause(), is(nullValue()));
+        assertThat(ex.getCode(), is("a0.sdk.internal_error.unknown"));
+        assertThat(ex.getStatusCode(), is(0));
+        assertThat(ex.getDescription(), is(any(String.class)));
```

Comment:
`is(equalTo("message"))` might be better so we test that the `.from(message)` actually sets/uses the correct message?
also the other test does this for the `from(string, int)` version